flit/upload.py

```diff
@@ -124,7 +126,7 @@ def get_repository(name=None, cfg_file="~/.pypirc"):
 
     return repo
 
-def write_pypirc(repo, file="~/.pypirc"):
+def write_pypirc(repo, file):
```

Comment:
You've changed the signature here, but not updated where the function is called (in `get_repository`).